apps/session-manager/src/repositories/__tests__/lollipop-revoke-queue.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { describe, test, expect, beforeEach, vi } from "vitest";
+import * as TE from "fp-ts/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/Either";
+import { RevokeAssertionRefInfo } from "@pagopa/io-functions-commons/dist/src/entities/revoke_assertion_ref_info";
+import {
+  mockQueueClient,
+  mockSendMessage,
+} from "../../__mocks__/repositories/queue-client.mocks";
+
+import { revokePreviousAssertionRef } from "../lollipop-revoke-queue";
+import { anAssertionRef } from "../../__mocks__/lollipop.mocks";
+import { base64EncodeObject } from "../../utils/encoding";
+
+describe("LollipopRevokeRepo#revokePreviousAssertionRef", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+  const expectedMessage = base64EncodeObject(
+    RevokeAssertionRefInfo.encode({ assertion_ref: anAssertionRef }),
+  );
+  test("should enqueue the message and return the expected response", async () => {
+    const expectedResponse = {
+      messageId: "aMessageId",
+      requestId: "aRequestId",
+    };
+    mockSendMessage.mockResolvedValueOnce(expectedResponse);
+    const result = await pipe(
+      revokePreviousAssertionRef(anAssertionRef)({
+        lollipopRevokeQueueClient: mockQueueClient,
+      }),
+      TE.map((value) => expect(value).toEqual(expectedResponse)),
+    )();
+    expect(mockSendMessage).toBeCalledWith(expectedMessage);
+    expect(E.isRight(result)).toBeTruthy();
```

Comment:
nitpick: we could define a `mockedDependencies` object, and we could check the result directly

```suggestion
    const mockedDependencies = {
      lollipopRevokeQueueClient: mockQueueClient,
    };
    const result = await pipe(
      mockedDependencies,
      revokePreviousAssertionRef(anAssertionRef),
    )();
    expect(mockSendMessage).toBeCalledWith(expectedMessage);
    expect(result).toEqual(E.right(expectedResponse));
```

Reply:
Fixed in 21e33f0